aio/deploy/helm-chart/kubernetes-dashboard/templates/configmap.yaml

```diff
@@ -18,3 +18,12 @@ metadata:
   labels:
 {{ include "kubernetes-dashboard.labels" . | indent 4 }}
   name: {{ template "kubernetes-dashboard.fullname" . }}-settings
+data:
+  _global: |
+    {
```

Comment:
Should we list all settings like you do or should the user be responsible for setting it?
For example:
```
{{ toYaml .Values.dashboardSettings }}
```
?

Reply:
That actually makes more sense. Just one concern - I'm not sure of the behaviour if extra/invalid args are passed. If that's not an issue, I would preset the toYaml approach.

Reply:
I wouldn't list all possible values. It's better to provide yaml as suggested by @desaintmartin.

Reply:
I would document an example of values in comments in values.yaml, but let the user be responsible for its settings. I'm actually concerned about updating/maintaining all possible settings.

Reply:
Also, I noticed that the `_global` key in CM `kubernetes-dashboard-settings` is a string of JSON.

Reply:
Oh, so that would be a multiline-string in values I suppose.

Reply:
changed it to a multiline string of YAML values